client/my-sites/email/form/mailboxes/components/mailbox-field/index.tsx

```diff
@@ -141,7 +151,15 @@ const MailboxField = ( {
 				/>
 				{ children }
 			</FormLabel>
-			{ field.error && <FormInputValidation text={ field.error } isError /> }
+			{ field.error && (
+				<FormInputValidation
+					// NTS: Passing markup as a 'text' prop feels odd.
+					text={ field.error.map( ( error, index ) => (
```

Comment:
I don't love that I'm passing this markup to a `text` prop, but it's typed as a ReactNode, so it works from that point of view. I considered passing the errors as `children`, but `text` is required, so we'd need something for that prop.

We could go down the road of:
- if a single error, pass it to `text`.
- if multiple errors, pass a different string like "Please address the following password issues" and then pass the errors as children. Essentially we'd be using `text` as a label.

Or we could look at the typing/naming of `FormInputValidation`'s props, but that feels like definite scope creep.